docs/code-examples/quick_start_connect.cpp

```diff
@@ -1,38 +1,17 @@
-#include <cmath>
-
 #include <rerun.hpp>
+#include <rerun/demo_utils.hpp>
 
-float remap(float value, float from_min, float from_max, float to_min, float to_max) {
-    return to_min + (to_max - to_min) * (value - from_min) / (from_max - from_min);
-}
+using namespace rerun::demo;
```

Comment:
I forget - is there a way to import just one name from a namespace?

```suggestion
using rerun::demo::grid;
```

Reply:
turns out this actually works just as you typed it. Didn't know either. Nice :D